README.md

```diff
@@ -454,18 +454,19 @@ This is a case of fingerprinting that is not user-hostile, and therefore one we
 would like to preserve.  With UA-CH this will be initially enabled by active
 collection of the various hints.
 
-This will work until something like [Privacy
-Budget](https://github.com/bslassey/privacy-budget) will prevent sites from
-collecting user identifying levels of entropy about their users.  By then, one
-can hope that alternative methods will exist for spam filtering that will
-obsolete the use of fingerprinting for that purpose.
+We hope that alternative methods or APIs will exist in the future to address the
```

Comment:
Two "in the futures".

Reply:
That's so you know it's "future-future"

(thanks)